devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusDev.java

```diff
@@ -86,11 +85,11 @@ public void setDebug(String debug) {
     }
 
     @OutputDirectory
+    @Optional
```

Comment:
why was this made optional? You really don't need one?

Reply:
Actually, thanks for catching this. It would fail if I did simply `./gradlew quarkusDev` complaining that property was not configured. Making it optional made it work. Do you think I should remove it?

Reply:
BTW, according to [1] "This annotation should be attached to the getter method in Java or the property in Groovy. Annotations on setters or just the field in Java are ignored."

I should probably fix it in other places.

[1] https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/Optional.html